src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -88,8 +88,8 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <!-- Visual Studio ships with some, but not all, of the assemblies in Microsoft.Composition, but we need them all -->
```

Comment:
Do we need this in both VisualStudioSetup *and* .Next?  I would have thought VisualStudioSetup was enough.